core/src/main/java/io/questdb/std/str/Utf8s.java

```diff
@@ -1049,11 +1049,15 @@ public static String stringFromUtf8Bytes(long lo, long hi) {
         if (hi == lo) {
             return "";
         }
-        Utf16Sink b = getThreadLocalSink();
-        if (!utf8ToUtf16(lo, hi, b)) {
-            throw CairoException.nonCritical().put("cannot convert invalid UTF-8 sequence to UTF-16");
+        Utf16Sink r = getThreadLocalSink();
+        if (!utf8ToUtf16(lo, hi, r)) {
+            Utf8StringSink sink = getThreadLocalUtf8Sink();
+            CairoException ex = CairoException.nonCritical().put("cannot convert invalid UTF-8 sequence to UTF-16 [seq=");
+            putSafe(lo, hi, sink);
```

Comment:
I'm not sure this would help to find the issue you're fixing here. What would really help is if we log the stack trace of this exception in ILP http. This is useful, you can keep this, but can you also please add stack trace logging of `CairoException` when assertions are enabled?

Reply:
My thought is to compare the message printed here with the original `ILP` message. This would help us determine whether the issue is with the field value or with memory corruption.